code/template-arguments-named/octane.handlebars

```diff
@@ -1 +1 @@
-<MyComponent @avatar="aang" />
\ No newline at end of file
+<MyComponent @answer=42 />
```

Comment:
This should be `@answer={{42}}`, right? at least that's what is suggested in `ember-template-lint`:

> Attribute [attribute] should be either quoted or wrapped in mustaches

Reply:
Good catch, thank you!